### PR TITLE
Update bunny

### PR DESCRIPTION
**Pull request by schlubbi**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 1.0.0 - 2015-10-28
```
Comment (duncan) [inline CHANGELOG.md]: :+1: for changelog

**Comment by netroy** [comment]
:+1:

File: watership.gemspec [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # coding: utf-8
-lib = File.expand_path('../lib', __FILE__)
+lib = File.expand_path("../lib", __FILE__)
```
Comment (duncan) [inline watership.gemspec]: What do you have against `'`? 😵

Comment (duncan) [inline watership.gemspec]: This was not a serious comment.

**Comment by duncan** [comment]
:+1:

**Comment by dschneider** [comment]
:+1:

**Pull request merged**